config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1585,50 +1585,52 @@ presubmits:
   #     Until then, this job complements pull-kubernetes-e2e-gce-alpha-features by running inplace pod resize e2e tests
   #     full-spectrum against containerd/main which has the necessary CRI support.
   - name: pull-kubernetes-e2e-inplace-pod-resize-containerd-main-v2
-    cluster: k8s-infra-prow-build
-    always_run: false
     optional: true
-    skip_report: false
+    always_run: false
     run_if_changed: '^(pkg\/kubelet\/kuberuntime\/|test\/e2e\/node/)'
-    skip_branches:
-      - release-\d+\.\d+  # per-release image
-    annotations:
-      testgrid-dashboards: sig-node-presubmits
-      testgrid-tab-name: pr-kubelet-gce-cluster-e2e-inplace-pod-resize-containerd-main-v2
-      description: Executes inplace pod resize e2e tests with containerd/main and cgroupv2
+    skip_report: false
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
       preset-dind-enabled: "true"
       preset-pull-kubernetes-e2e: "true"
       preset-pull-kubernetes-e2e-gce: "true"
+    annotations:
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pr-kubelet-gce-cluster-e2e-inplace-pod-resize-containerd-main-v2
+      description: Executes inplace pod resize e2e tests with containerd/main and cgroupv2
     spec:
       containers:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221109-489d560851-master
         args:
           - "--job=$(JOB_NAME)"
-          - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
-          - --service-account=/etc/service-account/service-account.json
```

Comment:
I think this needs to stay or else the job will not have service account to push the artifacts

Reply:
done. added it back